client/MentorPanel/src/redux/actions.js

```diff
@@ -45,18 +44,18 @@ export const loadQuestions = mentor_id => (dispatch) => {
         return questions
       }, {})
 
-      dispatch(loadTopics(mentor_id, questions))
+      dispatch(loadTopics(mentor_id, questions, recommended))
     }
   })
 }
 
-const loadTopics = (mentor_id, questions) => (dispatch) => {
+const loadTopics = (mentor_id, questions, recommended) => (dispatch) => {
```

Comment:
Since we use `Papa` a lot in the app now, I'm wondering if we can't make it work with `async` to make the code cleaner?

[this is def optional]

The basic idea would be create one wrapper for Papa that returns a `Promise`, e.g.


```
// from https://stackoverflow.com/questions/31375531/how-to-use-promises-with-papaparse
Papa.parseAsync = function(file) {
  return new Promise(function(complete, error) {
    Papa.parse(file, {complete, error});
  });
};
```

If had that could the below then be more like

```
const loadTopics = (mentor_id, questions, recommended) => async (dispatch) => {
    const topics_url = topicsUrl(mentor_id)
    try {
        result = await Papa.parseAsync(topics_url)
        ...
```